src/test/java/net/minecraftforge/debug/DynBucketTest.java

```diff
@@ -180,6 +162,29 @@ public void onBucketFill(FillBucketEvent event)
         }
     }
 
+    @Mod.EventBusSubscriber(value = Side.CLIENT, modid = MODID)
+    public static class ClientEventHandler
```

Comment:
I think it would be a lot cleaner to just `@SideOnly` the method instead of creating a new class.

That is:

```java
@SideOnly(Side.CLIENT)
@EventBusSubscriber(value = Side.CLIENT, modid = MODID)
public static void registerModels(ModelRegistryEvent mre) {
    ...
}
```